components/sysutils/nut/Makefile

```diff
@@ -0,0 +1,178 @@
+PKG_OPTIONS +=	-D NUTHOTPLUGDIR="$(NUTHOTPLUGDIR)"
+NUTUDEVDIR  =	$(strip .../,,...$(USRLIBDIR)/udev)
+PKG_OPTIONS +=	-D NUTUDEVDIR="$(NUTUDEVDIR)"
+
+### Recipe bits follow...
+COMPONENT_PREP_ACTION =	( cd $(SOURCE_DIR) && ./autogen.sh )
+
+COMPONENT_PRE_CONFIGURE_ACTION =	( $(CLONEY) $(SOURCE_DIR) $(@D) )
+
+CONFIGURE_ENV.32+=	LDFLAGS="$(LDFLAGS.32)"
+CONFIGURE_ENV.64+=	LDFLAGS="$(LDFLAGS.64)"
+
+CONFIGURE_OPTIONS.32+=	LDFLAGS="$(LDFLAGS.32)"
+CONFIGURE_OPTIONS.64+=	LDFLAGS="$(LDFLAGS.64)"
+
+CONFIGURE_OPTIONS +=	--disable-static
+CONFIGURE_OPTIONS +=	--enable-shared
+
+CONFIGURE_OPTIONS +=	--with-user=$(NUTUSER)
+CONFIGURE_OPTIONS +=	--with-group=$(NUTGROUP)
+CONFIGURE_OPTIONS +=	--sysconfdir=/$(NUTETCDIR)
+CONFIGURE_OPTIONS +=	--with-hotplug-dir=/$(NUTHOTPLUGDIR)
+CONFIGURE_OPTIONS +=	--with-udev-dir=/$(NUTUDEVDIR)
+CONFIGURE_OPTIONS +=	--with-statepath=/$(NUTSTATEDIR)
+CONFIGURE_OPTIONS +=	--with-systemdsystemunitdir=no
+CONFIGURE_OPTIONS +=	--with-dev
+CONFIGURE_OPTIONS +=	--with-libltdl
+CONFIGURE_OPTIONS +=	--with-ssl
+CONFIGURE_OPTIONS +=	--with-openssl
+
+CONFIGURE_OPTIONS.32 +=	--with-drvpath=/$(NUTDRVDIR)
+CONFIGURE_OPTIONS.32 +=	--with-cgipath=/$(NUTCGIDIR)
+CONFIGURE_OPTIONS.64 +=	--with-drvpath=/$(NUTDRVDIR)/$(MACH64)
+CONFIGURE_OPTIONS.64 +=	--with-cgipath=/$(NUTCGIDIR)/$(MACH64)
+
+# No separate toggle for this yet
+#CONFIGURE_OPTIONS +=	--with-nut-scanner=yes
+
+# Minimize guesswork: these are features we can and do build and deliver in
+# package manifest, so the build host must have relevant packages installed
+CONFIGURE_OPTIONS +=	--with-serial=yes
+CONFIGURE_OPTIONS +=	--with-usb=yes
+CONFIGURE_OPTIONS +=	--with-neon=yes
+CONFIGURE_OPTIONS +=	--with-avahi=yes
+CONFIGURE_OPTIONS +=	--with-powerman=yes
+CONFIGURE_OPTIONS +=	--with-cgi=yes
+
+# TODO: Revise NUT package when this gate provides (GNU) freeipmi >= 1.0.1
+# Note: we already have a p5m manifest for these products, just uncomment them
+CONFIGURE_OPTIONS +=	--with-ipmi=no
+CONFIGURE_OPTIONS +=	--with-freeipmi=no
+
+# Likely we'll not have this backend working any time soon
+CONFIGURE_OPTIONS +=	--with-linux_i2c=no
+CONFIGURE_OPTIONS +=	--with-macosx_ups=no
+
+# Note: Link with the correct ISA of net-snmp (for drivers) and gdlib (for cgi)
+CONFIGURE_OPTIONS +=	--with-snmp
+CONFIGURE_OPTIONS +=	--with-net-snmp-config=$(USRBINDIR)/net-snmp-config-$(BITS)
+
+CONFIGURE_OPTIONS.32 +=	--with-gdlib-config=$(USRBINDIR)/gdlib-config
+CONFIGURE_OPTIONS.64 +=	--with-gdlib-config=$(USRBINDIR64)/gdlib-config
+
+# Note: building the numerous documents of this projects is time-consuming
+# and we only need one copy. Also, we don't ship "dblatex" (needed here for
+# compiling PDFs) and probably don't require "html-chunked" chapters.
+CONFIGURE_OPTIONS.32 +=	--with-doc="man=yes"
+CONFIGURE_OPTIONS.64 +=	--with-doc="no"
+
+### Avoid occasionally possible hangs with chunked HTML doc, if we build it
+#CONFIGURE_OPTIONS +=	--with-doc="auto"
+
+CONFIGURE_OPTIONS +=	ASCIIDOC_VERBOSE="-v"
```

Comment:
Not exactly debug. If we would choose to build HTMLs (maybe later), the project comments suggest using this flag (verbosity), so I left this snippet here for future maintainers to keep it in mind. I am not sure if it does or not make a difference for just manpages as is the recipe content now.

I have not come to a strong opinion if we should build and deliver HTMLs (up-to-date versions are present on www.networkupstools.org anyway) or just the manpages would suffice. Source and content should be the same.